src/dependabot/verified_commits.ts

```diff
@@ -78,7 +78,7 @@ export async function getAlert (name: string, version: string, directory: string
 
   const nodes = alerts?.repository?.vulnerabilityAlerts?.nodes
   const found = nodes.find(a => (version === '' || a.vulnerableRequirements === `= ${version}`) &&
-      trimSlashes(a.vulnerableManifestPath) === `${trimSlashes(directory)}/${a.vulnerableManifestFilename}` &&
```

Comment:
@mwaddell without the propsed change, when `package.json` is at root, you get the following:
- `a.vulnerableManifestPath` equals `/package.json` which makex `trimSlashes(a.vulnerableManifestPath)` resolve to `package.json`
- `` `${trimSlashes(directory)}/${a.vulnerableManifestFilename}` `` is equivalent to `` `${trimSlashes('/')}/package.json` `` which yields `/package.json`

And so basically the slash trimmed paths never match, ending up making `getAlert` never return a matching security alert

Reply:
I understand now - thank you for the clarification